presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/alluxio/AlluxioHiveMetastore.java

```diff
@@ -32,6 +34,8 @@
 import com.facebook.presto.hive.metastore.thrift.ThriftMetastoreUtil;
```

Comment:
Is alluxio metastore a thrift-based metastore? If not, shall we restructure the util classes so that this class does not depend on `ThriftMetastoreUtil`? `MetastoreUtil` might be the right place to host some helpers.